app/javascript/packages/address-search/components/address-search.tsx

```diff
@@ -35,7 +35,7 @@ function AddressSearch({
         onFoundAddress={setFoundAddress}
         onFoundLocations={(locations) => {
           setLocationResults(locations);
-          onFoundLocations(locations);
+          onFoundLocations && onFoundLocations(locations);
```

Comment:
We should test that this prop is actually optional and include the type(s) in the function declaration.